### ISSUE TITLE
Secure scoring

### ISSUE THREAD

**Issue by kfcampbell** [DanceParty/MSHackGeoGuesser#45]
Sshhhhh....

**Comment by DanceParty** [comment]
Securing scoring requires setting up a backend, right?

**Comment by kfcampbell** [comment]
Yep! I'd like to do this work as I have some ideas for cheap architecture (you can have a pretty sophisticated server that runs entirely in Azure functions using .NET Core), but if someone else beats me to it, please let me know before you start so we can talk about it!
